genome.spec

```diff
@@ -181,36 +195,65 @@ typedef structure {
-RefSeq Reference - Reference, Representative and ExternalDB
-RefSeq Representative - Representative and ExternalDB
-RefSeq Latest or All Assemblies folder - ExternalDB
-User Data - User tagged
-
-Example Sources:
-RefSeq, Ensembl, Phytozome, Microcosm, User, RAST, Prokka, (other annotators)
-
+Field descriptions:
+    id: string - object id
+    scientific_name: string - human readable species name
+    domain: string - human readable phylogenetic domain name
+    warnings: list of string - genome-level warnings generated in the annotation process
+    genome_tiers: list of string - controlled vocabulary (based on app input and checked by the GFU)
+        Allowed values: Representative, Reference, ExternalDB, User
+        Tier assignments based on genome source:
+         * All phytozome - Representative and ExternalDB
+         * Phytozome flagship genomes - Reference, Representative and ExternalDB
+         * Ensembl - Representative and ExternalDB
+         * RefSeq Reference - Reference, Representative and ExternalDB
+         * RefSeq Representative - Representative and ExternalDB
+         * RefSeq Latest or All Assemblies folder - ExternalDB
+         * User Data - User tagged
+    feature_counts: map of string to integer - total counts of each type of feature
+        (key examples: "gene", "CDS", "mRNA", "non_coding_features")
+    genetic_code: int - An NCBI-assigned taxonomic category for the organism
+        See here: https://www.ncbi.nlm.nih.gov/Taxonomy/Utils/wprintgc.cgi 
+    dna_size: integer - total number of nucleotides
+    num_contigs: integer - total number of contigs in the genome
+    molecule_type: string - the type of molecule sequenced (eg. "DNA")
+    contig_lengths: list of int - nucleotide length of each contig in the genome
+    contig_ids: list of str - external database identifiers for each contig
+    source: str - controlled vocab - descriptor of where this data came from (eg. "RefSeq")
+        Allowed entries RefSeq, Ensembl, Phytozome, RAST, Prokka, User_upload
+    source_id: string - identifier of this genome from the source database (eg. the RefSeq ID such as "NC_000913")
+    md5: string - checksum of the underlying assembly sequence
+    taxonomy: string - semicolon-delimited taxonomy lineage, in order of parent to child
+    taxon_assignments: mapping of taxonomy namespace to taxon ID.
+        example: {"ncbi": "286", "gtdb": "s__staphylococcus_devriesei"}
+    gc_content: float - ratio of GC count to AT in the genome
+    publications: tuple of (pubmedid, source, title, web_addr, year, authors, journal). See typedef above.
+    ontology_events: A record of the service and method used for a set of
+        ontology assignments on the genome.
+    ontologies_present: a mapping of ontology source id (eg. "GO") to a mapping
+        of term IDs (eg "GO:16209") to term names or descriptions.
+    features: array of Feature - protein coding genes (see the separate Feature spec)
```

Comment:
Further documenting the distinction between feature and cds would be helpful e.g. mention introns

Reply:
Added a paragraph note below the field comments describing the difference between a feature and CDS as I understand it